movement-migration/validation-tool/src/checks/node/global_storage_includes.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::checks::error::ValidationError;
+use crate::types::storage::{MovementAptosStorage, MovementStorage};
```

Comment:
let's make changes in https://github.com/movementlabsxyz/aptos-core/tree/l1-migration.
For this validation, as discussed offline, you compare two DBs under the same latest aptos binary.